run.js

```diff
@@ -43,6 +43,18 @@ async function main() {
 
   // Sort the matches such that `utils` modules are compiled first,
   // because other packages may rely on them
+  // We also need to ensure that `now-client` is built before the CLI
+
+  matches.sort((a, b) => {
+    if (a.endsWith('-client') && !b.endsWith('-client')) {
+      return -1;
+    }
+    if (b.endsWith('-client') && !a.endsWith('-client')) {
```

Comment:
I'm not sure this will work. I think we need a single sort function with several `if` statements. Its probably best to put `now-cli` _last_ like this:


```js
  matches.sort((a, b) => {
    if (a.endsWith('-utils') && !b.endsWith('-utils')) {
      return -1;
    }
    if (b.endsWith('-utils') && !a.endsWith('-utils')) {
      return 1;
    }
    if (a === 'now-cli' && b !== 'now-cli') {
      return 1;
    }
    if (b === 'now-cli' && a !== 'now-cli') {
      return -1;
    }
    return b - a;
  });
```

Reply:
It did work though 😅🤔
I'll tweak it 🙏🏻